GTG/gtk/editor/taskview.py

```diff
@@ -43,8 +43,8 @@
 # are part of the word
 specials_separators = ['.', '/']
 
-bullet1_ltr = '→'
-bullet1_rtl = '←'
+bullet1_ltr = '→  '
+bullet1_rtl = '  ←'
```

Comment:
Won't this break if the user deletes the extra white spaces by mistake, or inserts their contents (with the mouse) inbetween the two whitespaces?

Reply:
Surprisingly no, you have to backspace at the first character of the task name to delete the the icon